react/player/src/asset/index.tsx

```diff
@@ -66,11 +66,16 @@ export const ReactAsset = (
       matchList.push(asset.key);
     });
 
-    const fuse = new Fuse(matchList, { keys: ["type", "key"] });
-    const similarType = JSON.stringify(
-      fuse.search(unwrapped.type as string)[0].item,
+    const typeList = matchList.map(
+      (match) => JSON.parse(JSON.stringify(match)).type,
     );
 
+    const similarType = typeList.reduce((prev, curr) => {
+      return leven(unwrapped.type, prev) < leven(unwrapped.type, curr)
```

Comment:
_nit_: can we store the value of the previous `leven` call, to duplication?

Reply:
````
    let minDistance = leven(unwrapped.type as string, typeList[0]);
    let similarType = typeList[0];
    typeList.forEach(type => {
      const distance = leven(unwrapped.type, type);
      if (distance < minDistance) {
        minDistance = distance;
        similarType = type;
      }
    });
````
when you say storing prev value, do you mean this?

Reply:
You can still keep everything in a single `reduce()`, but the way it's structured now we end up calling `leven()` for the previous values that we have already computed. 

something like: 
```ts
typeList.reduce((prev, curr) => {
  const next = {
    value: leven(unwrapped.type, curr),
    type: unwrapped.type
  };

  if (prev === undefined || prev.value < next.value) {
    return prev;
  }

  return next;
}, undefined);
```